.packcheck.ignore

```diff
@@ -1 +1,9 @@
 .packcheck.ignore
+stack.yaml
+.ci/stack-8.0.yaml
+.travis.yml
+.circleci/config.yml
+.github/workflows/packcheck.yml
+appveyor.yml
```

Comment:
This is a package for showing example configuration and for CI and it has ready made configs for popular CI platforms. So these files should be packaged in the distribution.

I think we should package all these files and even stack.yaml/cabal.project etc.